### PR TITLE
Updated index.html

### PR DESCRIPTION
**Pull request by arafattehsin**
Just add one point which I encountered lately during my career

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -42,6 +42,7 @@ <h2 class="title">A 1x Engineer...</h2>
         <li>Appreciates code reviews from their peers that are constructive and helpful.</li>
         <li>Is open to learning.</li>
         <li>Asks for help when they need it.</li>
+        <li>Listen to diffuse conflict and formulate better solutions.</li>
```
Comment (daneah) [inline docs/index.html]: For verb consistency! 😄 

```suggestion
        <li>Listens to diffuse conflict and formulate better solutions.</li>
```

Comment (bnb) [reply docs/index.html]: +1 to this. If you could merge @daneah's changes that would be swell.

**Comment by ccoenen** [comment]
I think it should be "defuse".

**Comment by arafattehsin** [comment]
100% correct. I'll change it and re-submit the PR.

**Comment by arafattehsin** [comment]
Will re-submit the PR.

**Pull request closed without merging**